### ISSUE TITLE
Field data shape

### ISSUE THREAD

**Issue by Drewniacki** [swag-kaust/Transform2022_SelfSupervisedDenoising#4]
Hello,

First of all thank you very much for this resource! Great stuff.

I was playing around with it and found out that in Tutorial 2, if any of the dimensions of filed data (as in `field_data.shape`) is not divisible by 4, the network returns error on `ExpandingBlock.forward()` (line 66 of file `unet.py`): `x = torch.cat([x, skip_con_x], axis=1)` due to `x` and `skip_con_x` being diferent shape. I assume it has something to do with convolutional layers but I lack skills to get to the bottom of it.
